packages/actor-query-result-serialize-stats/lib/ActorQueryResultSerializeStats.ts

```diff
@@ -45,37 +46,54 @@ export class ActorQueryResultSerializeStats extends ActorQueryResultSerializeFix
     data.push(`${header}\n`);
   }
 
+  /**
+   * @deprecated Use {@link createStat} instead.
+   */
+  /* istanbul ignore next */
```

Comment:
I'd prefer not to have `istanbul ignore next` in the codebase.
I think it's safe to remove these methods, as they are not really meant to be part of the public API.
But if we keep them, we need to have a unit test for them.

Reply:
This is resolved, will PR to next/major once this is merged